.github/workflows/ci_reusable_wf.yml

```diff
@@ -0,0 +1,404 @@
+# You can find more information about the design decisions here:
+# https://github.com/xaynetwork/xayn_ai/pull/21
+
+name: Reusable Dev CI
```

Comment:
Will a reusable workflow inherit the permissions from the calling workflow?

Reply:
Yes, it will take the permissions from the calling workflow.